src/pages/InitPool/FormExample.tsx

```diff
@@ -0,0 +1,282 @@
+
+                <Button
+                    title={'Gradient'}
+                    action={() => console.log('Confirm')}
+                />
+                <Button
+                    title={'Flat'}
+                    action={() => console.log('Confirm')}
+                    flat
+                />
+                <Button
+                    title={'Thin'}
+                    action={() => console.log('Confirm')}
+                    flat
+                    thin
+                />
+                <Button
+                    title={'Disabled'}
+                    action={() => console.log('Confirm')}
+                    flat
+                    disabled
+                />
+            </FlexContainer>
+        );
+    }, [rangeWidthProps, rangePriceInfoProps, minMaxPriceProps]);
+
+    return (
+        <FlexContainer
+            width='100vw'
+            height='100vh'
+            justifyContent='center'
+            alignItems='flex-start'
+            className={styles.background}
+        >
+            <FlexContainer
+                background='dark1'
+                rounded
+                padding='16px'
+                textAlign='center'
+                flexDirection='column'
+                outline='accent5'
+                fullWidth
+                alignItems='center'
+                // This means that if it's greater than lg, use these styles
+                lg={{
+                    margin: '64px 0 0 0',
+                    outline: 'accent1',
+                    width: 'auto',
+                }}
+            >
+                <FlexContainer
+                    rounded
+                    background='dark1'
+                    position='relative'
+                    padding={'16px'}
+                    transition
+                    flexDirection='column'
+                >
+                    {/* Header */}
+                    <FlexContainer as='header' height='41px'>
+                        <p />
+                        <Text
+                            fontSize='header1'
+                            margin='auto'
+                            fontWeight={'200'}
+                        >
+                            Example Form
+                        </Text>
+                        <FlexContainer
+                            gap={8}
+                            alignItems='center'
+                            fontSize='header1'
+                        >
+                            <p></p>
+                            <p />
+                            <IoIosCheckmarkCircle />
+                        </FlexContainer>
+                    </FlexContainer>
+                    {/* Body */}
+                    <FlexContainer
+                        gap={16}
+                        justifyContent='space-around'
+                        flexDirection='column'
+                        maxWidth='500px'
+                        lg={{
```

Comment:
@Jadessu @emilydrakesmith @benwolski notice this responsive styling with the new components.  It functions like tailwind where the base styles apply to all breakpoints and then the tags `sm, md, lg, xl, xxl` all apply for that size and up.  So here, we say that it's vertical with `column` flex and then from `lg` and up it becomes horizontal with `row` flex.